ulid.go

```diff
@@ -48,6 +48,8 @@ A ULID is a 16 byte Universally Unique Lexicographically Sortable Identifier
 type ULID [16]byte
 
 var (
+	Nil ULID // empty ULID, all zeros
```

Comment:
A ULID that's all zeroes is AFAIK valid, and so isn't really accurately called a "nil" ULID. At a minimum, this should be called `Zero ULID`. Even then, I'm not sure what purpose it would serve. Any application that accepts a `ulid.ULID` shouldn't in general treat a `Zero` ULID differently than any other ULID. Do you have some use case that would indicate otherwise?

Reply:
+1 for naming, `Zero` (and maybe a `isZero()` getter as in time library) will be better. I've several usecases where ULIDs are not already set and I avoid a lot of using pointers in my domain structs.

Reply:
It's important to understand that `var Zero ulid.ULID` — which is ultimately a `[16]byte` of all-zeroes — is a completely valid ULID value, and cannot be treated as any kind of "nil" or otherwise invalid ULID value. If you want to distinguish invalid vs. valid ULID values, you cannot model them as specific values of the ulid.ULID type.

Reply:
I understand and agree completely. In fact, I don't have any use cases where a zero-filled ULID is valid (they all start well after timestamp 0). So you recommend using a pointer to be totally sure of initialization ?

Reply:
If your application wants to treat a zero value ULID as invalid, no problem! I'm just not sure about treating this specific ULID value as special in the actual module.